code/game/objects/items/ego_weapons/non_abnormality/streetlight.dm

```diff
@@ -0,0 +1,54 @@
+//Streetlight stuff, this is all Grade 7 teth stuff, leader is grade 6.
+//They're generic weapons for the refinery, which is fine
+/obj/item/ego_weapon/city/streetlight_greatsword
+	name = "streetlight greatsword"
+	desc = "A greatsword used by the streelight office."
+	icon_state = "streetlight_greatsword"
+	force = 38
+	attack_speed = 2
+	damtype = RED_DAMAGE
+	armortype = RED_DAMAGE
+	inhand_icon_state = "claymore"
+	lefthand_file = 'icons/mob/inhands/weapons/swords_lefthand.dmi'
+	righthand_file = 'icons/mob/inhands/weapons/swords_righthand.dmi'
+	hitsound = 'sound/weapons/bladeslice.ogg'
+	attack_verb_continuous = list("attacks", "slashes", "stabs", "slices", "tears", "lacerates", "rips", "dices", "cuts")
+	attack_verb_simple = list("attack", "slash", "stab", "slice", "tear", "lacerate", "rip", "dice", "cut")
+
+/obj/item/ego_weapon/city/streetlight_bat
+	name = "streetlight office bat"
+	desc = "A yellow and black bat used by the streetlight office."
+	icon_state = "streetlight_bat"
+	force = 30
+	attack_speed = 1.5
+	damtype = RED_DAMAGE
+	armortype = RED_DAMAGE
+	attack_verb_continuous = list("bashes", "crushes")
+	attack_verb_simple = list("bash", "crush")
```

Comment:
Damage difference between these two is roughly 1 DPS, top should get a bit more damage as the latter has the knockback benefit attached and this one has none.

Reply:
Slower weapons is significantly better due to hit and run (Also These have been in the game forever I'm just moving them to their own file)